packages/infoblox_threat_defense/data_stream/event/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,195 @@
+        - infoblox_threat_defense.event.device.vendor
+        - infoblox_threat_defense.event.device.version
+        - infoblox_threat_defense.event.infoblox.b1.region
+        - infoblox_threat_defense.event.infoblox.b1.src_os_version
+        - infoblox_threat_defense.event.infoblox.b1.threat.indicator
+        - infoblox_threat_defense.event.infoblox.dns_q.class
+        - infoblox_threat_defense.event.infoblox.dns_q.type
+        - infoblox_threat_defense.event.infoblox.dns_r_code
+        - infoblox_threat_defense.event.infoblox.event.occurred_time
+        - infoblox_threat_defense.event.infoblox.host_id
+        - infoblox_threat_defense.event.infoblox.insight.id
+        - infoblox_threat_defense.event.infoblox.lease.op
+        - infoblox_threat_defense.event.infoblox.on_prem_host_name
+        - infoblox_threat_defense.event.infoblox.resource.id
+        - infoblox_threat_defense.event.infoblox.rpz.rule
+        - infoblox_threat_defense.event.infoblox.service_id
+        - infoblox_threat_defense.event.message
+        - infoblox_threat_defense.event.severity
+        - infoblox_threat_defense.event.source.address
+        - infoblox_threat_defense.event.source.mac_address
+        - infoblox_threat_defense.event.source.port
+        - infoblox_threat_defense.event.source.user_name
+        - infoblox_threat_defense.event.transport_protocol
+      tag: remove_custom_duplicate_fields
+      ignore_missing: true
+      if: ctx.tags == null || !ctx.tags.contains('preserve_duplicate_custom_fields')
+  - remove:
+      field: cef
+      tag: remove_cef
+      ignore_missing: true
+  - script:
+      tag: script_to_drop_null_values
+      lang: painless
+      description: This script processor iterates over the whole document to remove fields with null values.
+      source: |-
+        void handleMap(Map map) {
+          map.values().removeIf(v -> {
+            if (v instanceof Map) {
+              handleMap(v);
+            } else if (v instanceof List) {
+              handleList(v);
+            }
+            return v == null || v == '' || (v instanceof Map && v.size() == 0) || (v instanceof List && v.size() == 0)
+          });
+        }
+        void handleList(List list) {
+          list.removeIf(v -> {
+            if (v instanceof Map) {
+              handleMap(v);
+            } else if (v instanceof List) {
+              handleList(v);
+            }
+            return v == null || v == '' || (v instanceof Map && v.size() == 0) || (v instanceof List && v.size() == 0)
+          });
+        }
+        handleMap(ctx);
+  - set:
+      field: event.kind
+      tag: set_pipeline_error_into_event_kind
+      value: pipeline_error
+      if: ctx.error?.message != null
+  - append:
+      field: tags
+      value: preserve_original_event
+      allow_duplicates: false
+      if: ctx.error?.message != null
+on_failure:
+  - append:
+      field: error.message
+      value: |-
```

```suggestion
      value: >-
```
These should not preserve the new lines, we don't want them in the error messages that get rendered. (throughout)